.github/workflows/polyglot-chat-app.yml

```diff
@@ -36,4 +35,4 @@ jobs:
           ./gradlew build # pre-build application
           ./gradlew run &
           sleep 10
-          curl http://localhost:8080/
+          curl http://localhost:12345/
```

Comment:
@timfel , I think we should add the `./gradlew nativeCompile` check too.

Reply:
Not possible, the Github runners do not have enough RAM